### DISCUSSION TITLE
Add HTML Meta Tags

### DISCUSSION THREAD

**Discussion by jtechwyr** [kimai/kimai#4702] [Questions & Answers]
We want to add meta tags to the home page. In this case I guess it will be the login page additional html meta tags. How can this be done?

**Comment by kevinpapst** [comment]
There is no such feature. Workarounds might be:
- Using the Custom Content plugin, by injecting the tag via JS
- Patching the template directly

I cannot give support for any of those.

**Comment by jtechwyr** [comment]
Do you have instructions on how to edit the template?
